### DISCUSSION TITLE
More Throw Configurations

### DISCUSSION THREAD

**Discussion by bherdm** [rxhanson/RectanglePro-Community#524] [Ideas]
Hello,
I'm new to Pro, thank you for making this wonderful app! The throw configurations are amazing. I greatly wish for more slots or more configs. Would it be possible to set a custom throw+longthrow for each of the trackpad shortcuts? I'd like to have separate configs for 3 touches vs 4 touches.

**Comment by rxhanson** [comment]
Thanks for the feedback!

I have plans to build in some form of profiles, particularly for having different setups for different displays. I’ll think through having different profiles for the different triggers like this when I go down that path. 

As an aside, personally I use rest 4 for window throw and I have rest 5 configured in Charmstone (https://charmstone.app).
